### ISSUE TITLE
[Question] Is it possible to reduce the size of ESP box? certain object has a huge ESP box for some reason

### ISSUE THREAD

**Issue by TehChozinOne** [Akebi-Group/Akebi-GC#616]
title

**Comment by Wujian6T-T** [comment]
I also have this problem
Just wait 2 minutes

I don't know the specific reason

**Comment by TehChozinOne** [comment]
> panel "ESP-General-Min.Entity size" can be adjusted (**Please do more research before asking questions XD**)

yeah because it's clearly doesn't work. This is 0.100 entity size
![GenshinImpact_vOl5q9J6YS](https://user-images.githubusercontent.com/20630725/188426766-a84d587f-6acb-4527-9c72-7eff3b1b108e.png)

(**Please do more research before answering the question**)

**Comment by TehChozinOne** [comment]
> i didn't notice..."elemental monument after successful activation"

I don't know what were you trying to say here..
